modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-inventory-impl/src/main/java/com/liferay/headless/commerce/admin/inventory/internal/resource/v1_0/WarehouseOrderTypeResourceImpl.java

```diff
@@ -236,7 +236,7 @@ private WarehouseOrderType _toWarehouseOrderType(
 	private DTOConverterRegistry _dtoConverterRegistry;
 
 	@Reference(
-		target = "(component.name=com.liferay.headless.commerce.admin.inventoryinternal.dto.v1_0.WarehouseOrderTypeDTOConverter)"
+		target = "(component.name=com.liferay.headless.commerce.admin.inventory.internal.dto.v1_0.WarehouseOrderTypeDTOConverter)"
```

Comment:
Hey @RenanUmV, this change should not be here. Please fixup this with the proper commit. Thx